defaults/main.yml

```diff
@@ -30,3 +31,17 @@ confluence_jdbc_maven:
 confluence_system_dependencies:
   - virtualenv
   - python-setuptools
+
+confluence_catalina_opts: >
+  -Xloggc:$LOGBASEABS/logs/gc-`date +%F_%H-%M-%S`.log
+  -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 -XX:GCLogFileSize=2M
+  -XX:-PrintGCDetails -XX:+PrintGCDateStamps -XX:-PrintTenuringDistribution
+  -XX:G1ReservePercent=20
+  -Djava.awt.headless=true
+  -Datlassian.plugins.enable.wait=300
+  -Xms1024m -Xmx1024m -XX:+UseG1GC
```

Comment:
IMO, I would externalize the xms and xmx to different variables and then use them here